### PR TITLE
Multiple camera types support

### PR DESCRIPTION
**Pull request by dNechita**
Addresses #98

### CODE REVIEW

File: sdk/src/camera_factory.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#include <aditof/device_definitions.h>
+
+namespace aditof {
+
+Camera *CameraFactory::buildCamera(DeviceInterface *device) {
+    using namespace aditof;
+
+    DeviceDetails devDetails;
+    device->getDetails(devDetails);
+    switch (devDetails.sensorType) {
```
Comment (grigorovici-andreea) [inline sdk/src/camera_factory.cpp]: is this for future usage? as there is only 1 case, why switch?

File: sdk/src/target/usb_device.cpp [hunk 2]
```diff
@@ -77,3 +77,8 @@ UsbDevice::applyCalibrationToFrame(uint16_t * /*frame*/,
                                    const std::string & /*mode*/) {
     return aditof::Status::GENERIC_ERROR;
 }
+
+aditof::Status
+UsbDevice::getDetails(aditof::DeviceDetails & /*details*/) const {
```
Comment (grigorovici-andreea) [inline sdk/src/target/usb_device.cpp]: & /*details*/ 

Is this something I do not understand yet? or?

File: sdk/src/camera_factory.cpp (hunk 1 above)
Comment (dNechita) [reply sdk/src/camera_factory.cpp]: Yes, it is for future usage to support at least another type of camera.

File: sdk/src/target/usb_device.cpp (hunk 2 above)
Comment (dNechita) [reply sdk/src/target/usb_device.cpp]: This is a stub implementation so most function parameters are not used. We comment them so we don't get warnings about unused parameter. At some point we will treat warnings as errors.

**Pull request merged**